pkg/integration/components/paths.go

```diff
@@ -27,13 +27,6 @@ func (self Paths) ActualRepo() string {
 	return filepath.Join(self.Actual(), "repo")
 }
 
-// When an integration test first runs, we copy everything in the 'actual' directory,
-// and copy it into the 'expected' directory so that future runs can be compared
-// against what we expect.
-func (self Paths) Expected() string {
```

Comment:
This is a vestige of when we used snapshots. Now that we don't have an 'expected' path, it makes no sense to call the 'actual' path 'actual'. But I don't want to deal with that in this PR.